code/modules/shuttle/special.dm

```diff
@@ -145,23 +146,20 @@
 	laws = "1. Serve drinks.\n\
 		2. Talk to patrons.\n\
 		3. Don't get messed up in their affairs."
-	languages_spoken = ALL
-	languages_understood = ALL
 	status_flags = GODMODE // Please don't punch the barkeeper
 	unique_name = FALSE // disables the (123) number suffix
 
 /mob/living/simple_animal/drone/snowflake/bardrone/Initialize()
 	. = ..()
 	access_card.access |= access_cent_bar
+	grant_all_languages(polytongue=TRUE)
```

Comment:
You refer to polytongue in few places but the proc parameter is named omnitongue